subsys/mgmt/osdp/src/osdp_cp.c

```diff
@@ -729,117 +729,39 @@ int osdp_cp_set_callback_card_read(
 	return 0;
 }
 
-int osdp_cp_send_cmd_output(int pd, struct osdp_cmd_output *p)
+int osdp_cp_send_command(int pd, struct osdp_cmd *p)
```

Comment:
Use of these kind of short variable names like `p` here should be avoided (perhaps exception is the use of `i` in loops) as it does not give much hint for the usage of the variable. Not sure what it should be, perhaps containing `cmd` in the name.

Reply:
I will make this `struct osdp_cmd *cmd`. Thanks.